variants/github_actions_ci/template.rb

```diff
@@ -1 +1,3 @@
-directory("variants/github_actions_ci/workflows", ".github/workflows")
+template "variants/github_actions_ci/workflows/ci.yml.tt", ".github/workflows/ci.yml"
+copy_file "variants/github_actions_ci/workflows/deploy_to_ec2.yml", ".github/workflows/deploy_to_ec2.yml"
+copy_file "variants/github_actions_ci/workflows/deploy_to_heroku.yml", ".github/workflows/deploy_to_heroku.yml"
```

Comment:
@G-Rath Converted to Thor template (really ERB) because it'll let us generate cleaner YML (and will come in handy for future additions too I suspect)